src/navigation/navigators/SwipeNavigator.js

```diff
@@ -3,18 +3,28 @@ import React, { useEffect } from 'react';
 import { useSelector } from 'react-redux';
 
 import { ENABLE_NFTS } from '@/constants/nfts';
+import useDeepLink from '@/hooks/useDeepLink';
 import NFTs from '@/screens/tabs/NFTs';
 import ProfileScreen from '@/screens/tabs/Profile';
 import Tokens from '@/screens/tabs/Tokens';
+import { useNavigation } from '@/utils/navigation';
 
 import Routes from '../Routes';
 import BottomTabs from './BottomTabs';
 
 const Swipe = createMaterialTopTabNavigator();
 
-const SwipeNavigator = ({ route, navigation }) => {
+const SwipeNavigator = ({ route }) => {
   const { isInitialized, isUnlocked } = useSelector(state => state.keyring);
   const isLogin = route.name === Routes.LOGIN_SCREEN;
+  const { deepLink } = useDeepLink();
+  const navigation = useNavigation();
```

Comment:
why are we using this navigation object instead of the one received from params?

Reply:
if it doesnt work, try importing the useNavigation from react-navigation

import { useNavigation } from '@react-navigation/native';